tutorials/29_Serializing_Pipelines.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<p>grammar nit:</p><p> </p><p><code>that expects <strong><em>a</em></strong> topic from the user </code>
</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>